config/v1/types_ingress.go

```diff
@@ -90,8 +90,31 @@ type IngressSpec struct {
 	// provider of the current cluster and are required for Ingress Controller to work on OpenShift.
 	// +optional
 	LoadBalancer LoadBalancer `json:"loadBalancer,omitempty"`
+
+	// logLevel describes the desired logging verbosity for the Ingress Operator
+	// Any one of the following values may be specified:
+	// * Normal
+	// * Debug
+	// * Trace
+	// Defaults to "Normal".
+	// +optional
```

Comment:
```suggestion
	// +optional
        // +kubebuilder:default=Normal
```

Reply:
I think having the `+kubebuilder:default=Normal` flag in there was previously causing the integration test failure; it was causing `operatorLogLevel: Normal` to get inserted into the otherwise empty default config. That said, after removing the default flag, the description is wrong.

I'm not sure exactly what the common wording is for similar, but I'll update it to something like "When unset, the default log level is 'Normal'"

Reply:
Updating the integration test in this case is ok, adding a new field with a default is not a breaking change. This should be defaulted as it is then consistent with the existing operators which use this field